hack/test-update-storage-objects.sh

```diff
@@ -80,25 +84,50 @@ function cleanup() {
 
 trap cleanup EXIT SIGINT
 
-kube::etcd::start
+"${KUBE_ROOT}/hack/build-go.sh" cmd/kube-apiserver
 
-kube::log::status "Running test for update etcd object scenario"
+kube::etcd::start
 
-"${KUBE_ROOT}/hack/build-go.sh" cmd/kube-apiserver
+# need to include extensions/v1beta1 in new api version because its internal types are used by jobs
+# and hpas
+KUBE_OLD_API_VERSION="v1,extensions/v1beta1"
+KUBE_NEW_API_VERSION="v1,extensions/v1beta1,batch/v1,autoscaling/v1"
+KUBE_OLD_STORAGE_VERSIONS="batch=extensions/v1beta1,autoscaling=extensions/v1beta1"
+KUBE_NEW_STORAGE_VERSIONS="batch/v1,autoscaling/v1"
 
 
 #######################################################
 # Step 1: Start a server which supports both the old and new api versions,
 # but KUBE_OLD_API_VERSION is the latest (storage) version.
 #######################################################
-
 KUBE_API_VERSIONS="${KUBE_OLD_API_VERSION},${KUBE_NEW_API_VERSION}"
 RUNTIME_CONFIG="api/all=false,api/${KUBE_OLD_API_VERSION}=true,api/${KUBE_NEW_API_VERSION}=true"
-startApiServer
+startApiServer ${KUBE_OLD_STORAGE_VERSIONS}
+
+# Create object(s)
+
+# source_file,resource,namespace,name,old_version,new_version
+tests=(
```

Comment:
I think it will be better to move this above. Move this before startApiServer().